strawberry/enum.py

```diff
@@ -102,6 +102,12 @@ def _process_enum(
             deprecation_reason = item_value.deprecation_reason
             item_value = item_value.value
 
+            # TODO: this check should always be true, but if it's not we should
+            # raise an error (???)
+
+            if item.name not in cls._value2member_map_:
```

Comment:
this might not be true when doing something like:

@strawberry.enum
class TestEnum(Enum):
    A = "A"
    B = strawberry.enum_value("B")
    D = strawberry.enum_value("B")

Reply:
<img width="588" alt="CleanShot 2022-11-02 at 23 14 31@2x" src="https://user-images.githubusercontent.com/667029/199619800-6d979a46-cdbd-4a5d-b452-d32483b0396a.png">